src/pages/Signup/Signup.js

```diff
@@ -2,7 +2,108 @@ import React, { Component } from 'react';
 import './Signup.scss';
 
 export default class Signup extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+      repassword: '',
+      name: '',
+      smscheck: 'off',
+      emailcheck: 'off',
+    };
+  }
+
+  handleInput = e => {
+    const { name, value } = e.target;
+
+    this.setState({
+      [name]: value,
+    });
+  };
+
   render() {
-    return <div>Hello, Signup!</div>;
+    return (
+      <div className="signup">
+        <div className="signup-navbar"></div>
```

Comment:
```suggestion
        <nav className="signup-navbar"></nav>
```
기왕 nav 요소 만드는거 시맨틱 태그 사용해주는게 좋겠습니다.

Reply:
nav는 다른 분이 작업중이신 공통 컴포넌트로 받을 예정이라, 우선 레이아웃을 짜기 위해 동일한 height로 넣어두었습니다. 컴포넌트 완성되면 import하겠습니다.